CONTRIBUTING.md

```diff
@@ -13,3 +13,18 @@ Here are a few guidelines to follow:
 - **Write tests** to prevent regressions, unit tests with [Jest](https://jestjs.io/) in `__tests__` folder, end to end tests with [Playwright](https://playwright.dev/) in `e2e` folder
 
 If you happen to use VS Code, install the recommended extensions to get automatic formatting and linting on save, they are listed in `.vscode/extensions.json` and VS Code will prompt you to install them.
+
+In order to contribute, you need to set up the development envirment. 
+Use the following set up section for instructions.
```

Comment:
```suggestion
```

I think we can remove it. The "Set up" title says it all. If not removing then fixing the word envirment to environment.